webpack.config.ts

```diff
@@ -0,0 +1,28 @@
+import webpack from 'webpack';
+import { BuildMode, BuildPaths } from './config/webpack/types/types';
```

Comment:
The same for imports order Let's put internal imports after imports from 3rd parties